testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication.py

```diff
@@ -1077,13 +1077,11 @@ def CBL_offline_test(params_from_base_test_setup, sg_conf_name, num_of_docs):
         raise Exception("{0} failed".format(command))
 
     # 4. Do updates on CBL
-    time.sleep(180)
     db.update_bulk_docs(cbl_db, number_of_updates=number_of_updates)
     replicator.wait_until_replicator_idle(repl)
     replicator.stop(repl)
 
     # 6. CBL comes online( unblock ports)
-    time.sleep(180)
```

Comment:
as this test is not prefexed with test_, were u able to run test locally. This will run apple sccript . I think sleep is added to give sometime to take an effect.  As there is no alternative added for this sleep, but sleep removed , is test passed?